dev/sparktestsupport/modules.py

```diff
@@ -655,6 +655,7 @@ def __hash__(self):
         "pyspark.ml.tests.test_wrapper",
         "pyspark.ml.torch.tests.test_distributor",
         "pyspark.ml.torch.tests.test_log_communication",
+        "pyspark.ml.tests.connect.test_connect_function",
```

Comment:
Should also add `pyspark.ml.connect.functions`

Reply:
good catch!

I am thinking whether we should put the tests into `pyspark-connect`